manifests/installation-script

```diff
@@ -295,4 +295,4 @@ log("executed lens setup");
 
 ## Applying Housekeeping Job
 appHousekeeping = kubectl apply -n devtroncd devtronHousekeeping -u devtronHousekeepingOverride;
-log("executed devtron-housekeeping setup");
\ No newline at end of file
+log("executed devtron-housekeeping setup");
```

Comment:
Wasn't this to be applied from helm ?

Reply:
didn't change anything in this release for housekeeping job
It was applied using inception intentionally at the end to label and annotate all resources which are even created by inception so that we don't have to upgrade using CLI if we migrate other microservices in helm